Automattic/wp-calypso
# Domains: New inbound transfer domain status design

#### Changes proposed in this Pull Request

This PR introduces the new design for inbound transfer domains.

Transfer has not started yet:

<img width="728" alt="Screenshot 2020-03-27 at 12 27 02 PM" src="https://user-images.githubusercontent.com/13062352/77751597-4b24ea00-7026-11ea-8a24-3e2eae0e416f.png">

Transfer in progress:

<img width="728" alt="Screenshot 2020-03-27 at 12 29 19 PM" src="https://user-images.githubusercontent.com/13062352/77751782-a1922880-7026-11ea-8429-79ea48552b1d.png">

Transfer failed:

<img width="726" alt="Screenshot 2020-03-27 at 12 32 01 PM" src="https://user-images.githubusercontent.com/13062352/77751971-f7ff6700-7026-11ea-9c2f-1c63a77b2a6a.png">

#### Testing instructions

Try to transfer a domain (or by hacking the backend), and ensure that the proper view is shown based on the domain's transfer status.

NOTE: The pending status (first screenshot above) is shown when transfer is purchased from NUX
